eng/Packages.Data.props

```diff
@@ -86,7 +86,7 @@
 
     <!-- BCL packages -->
     <PackageReference Update="System.Buffers" Version="4.5.1" />
-    <PackageReference Update="System.ClientModel" Version="1.2.1" />
+    <PackageReference Update="System.ClientModel" Version="1.4.0-beta.1" />
```

Comment:
Do we want the global dependency to be a beta release?

Reply:
@jsquire Just want to confirm, is it okay to take the global dependency as a beta release, or should we stick to GA?

Reply:
We should probably leave it as a GA and then `VersionOverride` where the beta features are needed.